apps/prairielearn/src/ee/models/enrollment.test.ts

```diff
@@ -50,7 +50,11 @@ describe('getEnrollmentCountsForInstitution', () => {
       },
       CourseInstanceSchema,
     );
-    const firstCourseInstance = await selectCourseInstanceById('1');
+    const firstCourseInstance = await selectCourseInstanceById({
+      id: '1',
+      requestedRole: 'System',
+      authzData: dangerousFullSystemAuthz(),
+    });
```

Comment:
Discussed briefly with @reteps on Slack: should we consider test-only `select...` functions that take an ID and give us a row without any extra ceremony? cc @mwest1066 curious to hear your thoughts.